### PR TITLE
delete all from database

### PR DESCRIPTION
**Pull request by hbertrand**

### CODE REVIEW

File: buster/documents_manager/service.py [hunk 1]
```diff
@@ -76,3 +80,14 @@ def delete_source(self, source: str) -> tuple[int, int]:
         self.index.delete(filter={"source": source}, namespace=self.namespace)
 
         return source_deleted, documents_deleted
+
+    def drop_db(self):
```
Comment (jerpint) [inline buster/documents_manager/service.py:84]: can we add an `input()` where user has to explicitly agree to a destructive action? We should make it clear that this is not revertable

Comment (hbertrand) [reply buster/documents_manager/service.py:84]: done

File: buster/documents_manager/service.py [hunk 2]
```diff
@@ -76,3 +80,19 @@ def delete_source(self, source: str) -> tuple[int, int]:
...
+        """Drop the currently accessible database.
+
+        For Pinecone, this means deleting everything in the namespace.
+        For Mongo DB, this means dropping the database. However this needs to be done manually through the GUI.
+        """
+        confirmation = input("Dropping the database is irreversible. Are you sure you want to proceed? (Y/n): ")
```
Comment (jerpint-mila) [inline buster/documents_manager/service.py:90]: Nitpick , but it should be N/y since No is by default

Comment (hbertrand) [reply buster/documents_manager/service.py]: done

**Review by jerpint — APPROVED** [review]

**Pull request merged**